ownership-chain/runtime/laos/src/tests/mod.rs

```diff
@@ -142,49 +142,3 @@ fn account_vests_correctly_over_time() {
 		assert_eq!(Balances::usable_balance(&bob), total_vested_amount);
 	});
 }
-
-#[test]
-fn staking_inflation_rewards_is_deactivated_by_default() {
-	new_test_ext().execute_with(|| assert!(ParachainStaking::inflation_enabled() == false));
-}
-
-#[test]
-fn collator_rewards_account_is_not_set_by_default() {
-	new_test_ext().execute_with(|| assert!(ParachainStaking::collator_rewards_account().is_none()));
-}
-
-#[test]
-fn fees_go_to_collator_rewards_account() {
```

Comment:
what does happen with fees now?

Reply:
I am implementing it here : https://github.com/freeverseio/laos/pull/408